endogenous/scripts/end_transport.py

```diff
@@ -153,7 +211,7 @@ def transport(string):
         bus0="electricity bus",                               
```

Comment:
there should be two different links "ICE vehicle " and "EV"

Reply:
Link with ICE vehicle is from line 137/193 to 148/206
Link with EV starts from line 150/208
you can see it, if you expand the code in between.
Or do you mean something different?

Reply:
You are right, I read it to fast, forget about it. I mark this point as resolved.